Lib/test/test_tokenize.py

```diff
@@ -1351,6 +1351,28 @@ def assertExactTypeEqual(self, opstr, *optypes):
         self.assertEqual(tok_name[tokens[1 + num_optypes].exact_type],
                          tok_name[token.ENDMARKER])
 
+    def test_all_literal_tokens(self):
+        non_literals = {
+            token.ENDMARKER, token.NAME, token.NUMBER,
+            token.STRING, token.NEWLINE, token.INDENT,
+            token.DEDENT, token.OP, token.ERRORTOKEN,
+            token.AWAIT, token.ASYNC,
+            token.COMMENT, token.NL, token.ENCODING,
+            token.N_TOKENS, token.NT_OFFSET,
+        }
+
+        for tok in range(token.N_TOKENS):
+            if tok in non_literals:
+                continue
+            name = tok_name[tok]
+            with self.subTest(name=name):
+                self.assertIn(tok, EXACT_TOKEN_TYPES.values())
```

Comment:
Looking at [CI logs](https://travis-ci.org/python/cpython/jobs/246655056) of an earlier commit, I wonder if we should make the following message clearer:

```
======================================================================
FAIL: test_all_literal_tokens (test.test_tokenize.TestTokenize) (name='NL')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/python/cpython/Lib/test/test_tokenize.py", line 1368, in test_all_literal_tokens
    self.assertIn(tok, EXACT_TOKEN_TYPES.values())
AssertionError: 58 not found in dict_values([7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 52, 51, 49, 50])
```